app/src/main/java/com/mensinator/app/business/IClueImport.kt

```diff
@@ -1,6 +1,5 @@
 package com.mensinator.app.business
 
 interface IClueImport {
-    fun getDefaultImportFilePath(): String
-    fun importFileToDatabase(filePath: String)
+    fun importFileToDatabase(filePath: String) : Boolean
```

Comment:
Suggestion: Add this explanation text. Will show up like this:

```
    /**
     * Read the content of the given file and attempts to import the data into the database.
     *
     * @return true if the import was successful, false otherwise
     */
```

![studio64_bcHW7t0cuF](https://github.com/user-attachments/assets/09f4a0cd-ad55-4c86-8da4-9fdfb24ad5ab)

Reply:
Also to IFloImport